### PR TITLE
Add information about time series content to file name

### LINKED ISSUE #214: Integrate column scheme in time series file name
Until now, the time series file name does not reveal any information about it's content. To know, if you get a whole sale market price time series or an active power time series, you have to open the file.

To aid human file handling and simplify the code based file handling, integrate the column scheme in the time series file name.

### PR DESCRIPTION
**Pull request by ckittl**
Closes #214 ~, address #208 before, pls.~

### CODE REVIEW

**Comment by johanneshiry** [comment]
Would prefer merging #208 before to have less struggle with the javadocs which makes a careful review nearly impossible.

**Comment by ckittl** [comment]
> Would prefer merging #208 before to have less struggle with the javadocs which makes a careful review nearly impossible.

Did so, you are good to go. ;-)

**Comment by ckittl** [comment]
!test

File: docs/readthedocs/io/csvfiles.rst [hunk 1]
```diff
@@ -81,13 +81,48 @@ Input
...
+Those files are meant to carry different types of content - one might give information about wholesale market prices,
+the other is a record of power values provided by a real system.
+To be able to understand, what's inside of the file, the *columnScheme* part of the file name gives insight of it's
+content.
+The following keys are supported until now:
+
++---------+----------------------------------------------------------------------------------------------------------------+
+| Key     | Information and supported head line                                                                            |
++=========+================================================================================================================+
+| c       | | An energy price (e.g. in €/MWh; c stands for charge".                                                        |
```
Comment (johanneshiry) [inline docs/readthedocs/io/csvfiles.rst:102]:
```suggestion
| c       | | An energy price (e.g. in €/MWh; c stands for charge).                                                        |
```

File: src/main/java/edu/ie3/datamodel/io/factory/EntityData.java [hunk 2]
```diff
@@ -36,8 +36,8 @@
   private static final Logger logger = LoggerFactory.getLogger(EntityData.class);
   private static final GeoJsonReader geoJsonReader = new GeoJsonReader();
 
-  private final Map<String, String> fieldsToAttributes;
-  private final Class<? extends UniqueEntity> entityClass;
+  protected final Map<String, String> fieldsToAttributes;
```
Comment (johanneshiry) [inline src/main/java/edu/ie3/datamodel/io/factory/EntityData.java:39]: Nope
```suggestion
  pivate final Map<String, String> fieldsToAttributes;
```

use `getFieldsToValues()` instead

File: src/main/java/edu/ie3/datamodel/io/factory/EntityData.java [hunk 3]
```diff
@@ -36,8 +36,8 @@
...
+  protected final Class<? extends UniqueEntity> entityClass;
```
Comment (johanneshiry) [inline src/main/java/edu/ie3/datamodel/io/factory/EntityData.java:40]:
```suggestion
 private  final Class<? extends UniqueEntity> entityClass;
```

**Review by johanneshiry — CHANGES_REQUESTED** [review]
Generally LGTM - pls address codacy issues + the minor changes I commented

**Pull request merged**